modules/@apostrophecms/page/index.js

```diff
@@ -3233,24 +3233,35 @@ database.`);
           });
         });
       },
-      async inferLastTargetIdAndPosition(doc) {
+      async inferLastTargetIdAndPosition(doc, { publishedTargetsOnly = false } = {}) {
         const parentPath = self.getParentPath(doc);
         const parentAposDocId = parentPath.split('/').pop();
         const parentId = doc.aposLocale
           ? `${parentAposDocId}:${doc.aposLocale}`
           : parentAposDocId;
+        if (publishedTargetsOnly) {
+          const parent = await self.apos.doc.db.findOne({ _id: parentId });
```

Comment:
Looks like `parentId` can also be an `aposDocId` if doc has no `aposLocale`, it might happen only on a non localized site? I still don't think this request would be valid.
Also, are we sure it will request the draft version of the parent? It's ok if we're sure doc is always draft.